design/FY2017/MultipleAirloopsInOneZone.md

```diff
@@ -0,0 +1,213 @@
+Allow Multiple Air Loops to One Thermal Zone
```

Comment:
Mike,

This is long overdue :) Thanks for your effort on this new feature.

I have several questions:

1. Do we allow multiple outdoor air loops (DOAS systems) to a zone?

2. How do we control the operation of multiple air loops? For example, if we have two air loops to a zone with each sized to meet half of the zone peak loads, at 50% zone load, do we operate only one air loop at 100%, or two with each at 50%? We may need a control scheme for this to make it more flexible. 

3. How to avoid fight of different air loops (e..g, some doing cooling/dehumidifying while others doing heating/humidifying)? We saw this in data centers with multiple RTUs.

4. Any high level report to summarize how various air loops operate in a zone? E.g., what loads they meet, how many hours each loop operate?

Tianzhen